src/fun/dice/DiceCommand.ts

```diff
@@ -0,0 +1,55 @@
+import Command from "../../core/Command";
+import { Message } from "discord.js";
+import { createBaseEmbed } from "../../util/Messages";
+import Store from "../../core/Store";
+import Constants from "../../core/Constants";
+
+class DiceCommand extends Command {
+    constructor(store: Store, prefix: string) {
+        super(store, prefix, "dice", 1);
+    }
+
+    checkPermission(msg: Message) {
+        return !msg.author.bot;
+    }
+
+    async execute(msg: Message, args: string[]) {
+        if (args.length === 0) {
+            var rand = Math.floor(Math.random() * 6) + 1;
+            await msg.channel.send(createBaseEmbed("You rolled a " + rand, Constants.black));
+        } else if (args.length === 1 && args[0] === "help"){
+            await msg.channel.send(createBaseEmbed(`Available commands are:
+\`-dice\` - roll a 6-sided dice
+\`-dice <sides>\` - roll a dice with a specific amount of sides
+\`-dice <stuff>\` - choose randomly from a list of comma separated stuff
+\`-dice <min>-<max>\` - choose randomly between numbers in a range
+\`-dice help\` - show this help message`, Constants.black));
+        } else if (args.length === 1 && !isNaN(parseFloat(args[0]))){
```

Comment:
I'm a bit late on this reply, but the only change left to make here is change this line to
`} else if (args.length === 1 && !isNaN(parseFloat(args[0])) && parseFloat(args[0]).toString() === args[0]) {`
parseFloat is just a bit too good at parsing and even if I enter "1-5" it parses it as 1. This should fix that problem. All the other stuff I've tested and works perfectly, and I'll just match style and formatting when it's fully merged in!

Reply:
Okay, thanks. I think you can just push to my branch like you own it, but I'm not entirely sure.